lib/Pipfile

```diff
@@ -31,6 +31,7 @@ seaborn = "*"
 prometheus-client = "*"
 opencv-python = "*"
 requests-mock = "*"
+keras = "*"
```

Comment:
Do we really need keras?

Reply:
This is in the dev dependencies. Some of our examples use keras.